gci/linter_repos.py

```diff
@@ -0,0 +1,62 @@
+from multiprocessing import Pool
+
+import giturlparse
+
+from community.git import get_irepo
+
+
+def simplify_issue(issue, repo_url=''):
+    return {
+      'id': issue.number,
+      'repo_url': repo_url,
+      'title': issue.title,
+      'labels': list(issue.labels),
+      'description': issue.description,
+    }
+
+
+def _lower_set(iterable):
+    return set(map(str.lower, iterable))
+
+
+def issue_filter(issue, labels, desc_keywords):
+    norm_issue_labels = _lower_set(issue['labels'])
+    norm_labels = _lower_set(labels)
+
+    if len(norm_issue_labels & norm_labels) > 0:
+        return True
+
+    norm_keywords = _lower_set(desc_keywords)
+    norm_text = str.lower(issue['title'] + ' ' + issue['description'])
+    norm_text = set(norm_text.split())
+
+    if len(norm_keywords & norm_text) > 0:
+        return True
+
+    return False
+
+
+def fetch_issues(targets, labels=[], desc_keywords=[], threads=8, debug=False):
+    all_repo_issues = {}
+
+    for target in targets:
+        if debug:
+            print('processing', target)
+
+        parsed_url = giturlparse.parse(target)
+        repo = get_irepo(parsed_url)
+
+        with Pool(processes=threads) as pool:
```

Comment:
When I saw variable name `threads` I thought this was multithreading, but it turns out to be multiprocessing ^^

Reply:
I was about to use https://docs.python.org/dev/library/multiprocessing.html#module-multiprocessing.dummy.